### ISSUE TITLE
Fix building of org.openetcs.sysml2scade.transformation

### ISSUE THREAD

**Issue by jastram** [openETCS/toolchain#378]
The above project does not compile, and is not part of the maven build. To make it pass, I:
- [x] Removed the building of the datadictionary from ...parent/pom.xml 
- [x] Removed the datadictionary feature from the product

Now Maven builds again. To fix the underlying problem, please:
- [ ] Update the target platform, so that the transformation project compiles
- [ ] Add a pom.xml to the transformation plugin, to drive the maven build
- [ ] Add the transformation project to  ...parent/pom.xml 
- [ ] Once this is done, reactivate building the datadictionary
- [ ] Optionally, add the datadictionary back to the project.  There is a debate whether this should be done or not.
